lib/PuppeteerSharp/CustomQueriesManager.cs

```diff
@@ -235,7 +235,7 @@ private static PuppeteerQueryHandler CreatePuppeteerQueryHandler(CustomQueryHand
                     var jsHandle = await element.EvaluateFunctionHandleAsync(
```

Comment:
`handle` on line 234 and 283 are now unused

Reply:
thank you!